### ISSUE TITLE
Slitaz and Porteus

### ISSUE THREAD

**Issue by nobicycle** [IsaacSchemm/MultiCD#44]
> Copying SliTaz...
> cp: cannot stat '/media/esata/SYSTEM/tmp/MultiCD-git/temporary-mountpoints/slitaz/boot/bzImage': No such file or directory

Slitaz also exists as slitaz-rolling-core64.iso, containing bzImage64. Plugin may need updating.

> Copying Porteus...
> cp: cannot stat '/media/esata/SYSTEM/tmp/MultiCD-git/temporary-mountpoints/porteus/porteus': No such file or directory

Porteus Porteus-Kiosk-4.1.0-x86_64.iso no longer has this structure. Plugin may need updating.
 
Best wishes

**Comment by IsaacSchemm** [comment]
I updated the SliTaz plugin. Porteus will be more work, since the kiosk version has a different file tree structure than regular Porteus.

**Comment by nobicycle** [comment]
Thank you Isaac.
I should have been capable of contributing these updates but I found the entry level knowledge too difficult to acquire.  Maybe I am too lazy? :-)
I know you have an annotated example of a plugin, but perhaps a step by step procedure would help the lazy, like me.  If you agree I will post it as a new issue.
Best wishes

**Comment by IsaacSchemm** [comment]
The Porteus Kiosk fix was tricky. I had to edit a file in the initrd so it was looking for the correct ISO label, which by defaul is "MultiCD" (it was looking for something ending in "Kiosk"). I ended up uncompressing the initrd and running sed on it to replace the text.
Then the kernel wouldn't take my xz or gz compressed initrds, so I had to leave it uncompressed on the ISO.

Most things aren't as hard as that one. If you want to write a plugin, start out by manually building a live CD from the command line - I know there are some tutorials floating around.
